sopel/modules/reddit.py

```diff
@@ -113,20 +113,58 @@ def rpost_info(bot, trigger, match):
         return NOLIMIT
 
 
-# If you change this, you'll have to change some other things...
-@commands('redditor')
-@example('.redditor poem_for_your_sprog')
-def redditor_info(bot, trigger, match=None):
+def subreddit_info(bot, trigger, match, commanded=False):
```

Comment:
Instead of `commanded`, I'd rather see a `is_auto` parameter (which has the opposite meaning of course). If you want to keep the "commanded" approach, I'll then suggest to rename to `is_command`.

Reply:
going to go with `is_command` for this PR, to retain the current workings of all functions involved.

`reddit.py` is kinda a mess,, which is sad coming from me?